framework/src/play/mvc/Scope.java

```diff
@@ -30,7 +30,7 @@
     public static final String COOKIE_EXPIRE = Play.configuration.getProperty("application.session.maxAge");
     public static final boolean SESSION_HTTPONLY = Play.configuration.getProperty("application.session.httpOnly", "false").toLowerCase()
             .equals("true");
-    public static final boolean SESSION_SEND_ONLY_IF_CHANGED = Play.configuration
+    public static boolean SESSION_SEND_ONLY_IF_CHANGED = Play.configuration
```

Comment:
@tomparle is this change related to JDK13?

Reply:
Yes, see details in my original issue : https://github.com/playframework/play1/issues/1324

Reply:
gotcha, thx!